client/app/scripts/superdesk-archive/module.js

```diff
@@ -86,6 +86,9 @@ define([
                     }
                 }, function(response) {
                     item.error = response;
+                    if (angular.isDefined(response.data._issues['validator exception'])) {
```

Comment:
Better to check if `response.data._issues` is defined or not as in some failure cases this wouldn't be defined.